source/includes/steps-fle-convert-to-a-remote-master-key-gcp.yaml

```diff
@@ -116,9 +116,20 @@ content: |
         .. tab::
           :tabid: nodejs
 
+          .. code-block:: javascript
+
+             kmsProviders = {
+               gcp: {
+                 emai": "<GCP service account email>",
+                 privateKey: "<GCP service account private key>",
+                 endpoint: "<GCP authentication endpoint>",
```

Comment:
In the Azure examples, you omitted the trailing comma -- should this be omitted as well or should the Azure examples be updated to stay consistent?

Reply:
Thanks, I've updated throughout